lib/AST/ASTVerifier.cpp

```diff
@@ -988,6 +988,13 @@ class Verifier : public ASTWalker {
       }
       
       if (S->hasResult()) {
+        if (isa<ConstructorDecl>(func)) {
```

Comment:
I'd like to write a test for this but, I'm not sure how to. I don't think there's a way to read a textual AST into the compiler but, if so, maybe we could do that.

Reply:
ASTVerifier is not testable today :(